README.md

```diff
@@ -27,255 +20,169 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
-to use Deephaven. These data sets are used extensively in Deephaven's documentation and are needed to run some examples. [Deephaven's examples repository](https://github.com/deephaven/examples) contains documentation on the available data sets and how to manage them. 
+On Windows, all commands must be run inside a WSL 2 terminal.
 
-If you have chosen a deployment with example data, the example data sets will be downloaded. Production deployments containing your own data will not need the example data sets.
+:::
 
+### Authentication
 
-To upgrade a deployment to the latest example data, run:
+Deephaven, by default, uses pre-shared key authentication to authenticate against unauthorized access. For more information on supported authentication methods:
 
-```bash
-docker-compose run examples download
-```
+- [How to configure and use pre-shared key authentication](https://deephaven.io/core/docs/how-to-guides/authentication/auth-psk/)
+- [How to configure and use anonymous authentication](https://deephaven.io/core/docs/how-to-guides/authentication/auth-anon/)
+- [How to configure and use username/password authentication](https://deephaven.io/core/docs/how-to-guides/authentication/auth-uname-pw/)
+- [How to configure and use mTLS authentication](https://deephaven.io/core/docs/how-to-guides/authentication/auth-mtls/)
+- [How to configure and use Keycload/OpenID authentication](https://deephaven.io/core/docs/how-to-guides/authentication/auth-keycloak/)
 
-To see what other example data management commands are available, run:
+If using a randomly generated pre-shared key, you can find the key in the server logs:
 
 ```bash
-docker-compose run examples
+docker compose logs -f | grep "access through pre-shared key"
 ```
 
-If your deployment does not have example data, these commands will fail with `ERROR: No such service`.
-
-
-## Run Deephaven IDE
-
-Once Deephaven is running, you can launch a Deephaven IDE in your web browser.  Deephaven IDE allows you to interactively analyze data.
-
-- If Deephaven is running locally, navigate to [http://localhost:10000/ide/](http://localhost:10000/ide/).
-- If Deephaven is running remotely, navigate to `http://<hostname>:10000/ide/`, where `<hostname>` is the address of the machine Deephaven is running on.
+### Connect to the server
 
-![alt_text](docs/images/ide_startup.png "Deephaven IDE")
+Deephaven is run from a web browser, and can be connected to via `http://localhost:10000/ide`. If using authentication, enter credentials to gain access to the IDE.
 
-# First query
+## First query
 
 From the Deephaven IDE, you can perform your first query.
 
-This script creates two small tables: one for employees and one for departments.
+This Python script creates two small tables: one for employees and one for departments.
 It joins the two tables on the DeptID column to show the name of the department
 where each employee works.
 
 ```python
```

Comment:
do we also want to include the equivalent groovy query and include a note that they need to choose the script based on if their server is Python on Grooy?